backend/README.md

```diff
@@ -8,7 +8,6 @@ Documentation is accessible through Swagger.
 
 ## Accessing the API Documentation
 
-
 http://localhost:3000/api/api-docs
```

Comment:
Once up and running this should be a link to your OpenShift realm, if there is a permanent one 😏